kubernetes-client/src/test/java/io/fabric8/kubernetes/client/dsl/base/BaseOperationTest.java

```diff
@@ -21,8 +21,12 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
+<<<<<<< HEAD
+import static org.junit.jupiter.api.Assertions.assertEquals;
+=======
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+>>>>>>> d810aa1da99ff5cdb81cabea4f5a6dcf55ff0218
```

Comment:
This needs to be fixed, probably you forced a merge while rebasing.

Reply:
Thank you sir.